yubico-webauthn/1.3.0.wso2v1/pom.xml

```diff
@@ -81,6 +81,10 @@
                     <groupId>org.bouncycastle</groupId>
                     <artifactId>bcpkix-jdk15on</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.bouncycastle</groupId>
+                    <artifactId>bcprov-jdk15on</artifactId>
+                </exclusion>
```

Comment:
Is the requirement is to just remove from yubico bundle or exclude the library from the product itself. If it's to be removed from the product itself, it would be better to do this exclusion in the parent pom at the top level. WDYT?

Reply:
The requirement is to remove the bcprov jar file from bundling to the yubico-webauthn jar.